src/contracts/GPv2AccessControl.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+
+/// @title Gnosis Protocol v2 Access Control Contract
+/// @author Gnosis Developers
+contract GPv2AccessControl is Ownable {
```

Comment:
nit: I think it would be nice to define the critical method `isSolver` as an [Interface](https://docs.soliditylang.org/en/v0.6.12/contracts.html#interfaces) that this contract implements (as we could envision other implementations such as one hardcoded solver, etc.).

Reply:
Very nice idea. I will dig into this tomorrow!

Reply:
All done!